### ISSUE TITLE
Remember last window position when tray icon is clicked

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#56]
```
What steps will reproduce the problem?
1. Move pino to the right side of the screen
2. Click on the pino icon in the Notification area to hide Pino
3. Click on the pino icon again, and notice that the window is now on the 
left side of the screen

What is the expected output? What do you see instead?

Pino should remember where it was last placed and preserve that position.

What version of the product are you using? On what operating system?

Pino trunk, openSUSE 11.2
```

Original issue reported on code.google.com by `brad.tay...@gmail.com` on 25 Jan 2010 at 3:58

**Comment by GoogleCodeExporter** [comment]
```
Hmm, it is works for me.
```

Original comment by `tro...@gmail.com` on 31 Jan 2010 at 1:28

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem. But I always get stuck in the lower right corner for 
me.
```

Original comment by `58desi...@gmail.com` on 4 Feb 2010 at 5:41

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Same for me window position not saved. On both Ubuntu Lucid Lynx AMD64 and 
Karmic
Koala x86. Both running Pino 0.1.5.

Keep up the good work.
```

Original comment by `mez.pahlan` on 14 Feb 2010 at 3:48

**Comment by GoogleCodeExporter** [comment]
```
Same problem here too on Karmic 64bit.
```

Original comment by `bote...@gmail.com` on 17 Feb 2010 at 11:45

**Comment by GoogleCodeExporter** [comment]
```
Got Karmic 64bit and Pino 0.1.5 and it works fine!
```

Original comment by `aku...@gmail.com` on 18 Feb 2010 at 2:37

**Comment by GoogleCodeExporter** [comment]
```
Same problem

I have pino 0.2.0 via the PPA on Ubuntu Karmic Koala 9.10, 32bit.

It always appears on right-hand side of screen when launched or when un-hidden 
from tray.

FWIW, installed versions of dependencies are (via apt-cache depends pino):

 Depends: libatk1.0-0
  Depends: libc6
  Depends: libcairo2
  Depends: libdbus-1-3
  Depends: libdbus-glib-1-2
  Depends: libfontconfig1
  Depends: libfreetype6
  Depends: libgee2
  Depends: libglib2.0-0
  Depends: libgtk2.0-0
  Depends: libnotify1
  Depends: <libnotify1-gtk2.10>
    libnotify1
  Depends: libpango1.0-0
  Depends: libsoup2.4-1
  Depends: libunique-1.0-0
  Depends: libwebkit-1.0-2
  Depends: libxml2

```

Original comment by `jonathan...@gmail.com` on 26 Feb 2010 at 8:53

**Comment by GoogleCodeExporter** [comment]
```
Closed by 4801486fa0 revision.
```

Original comment by `tro...@gmail.com` on 28 Mar 2010 at 3:07
- Changed state: **Fixed**
